src/Property.php

```diff
@@ -143,6 +143,41 @@ protected function cast($value)
         return new $castTo($value);
     }
 
+    protected function castArray(array $values)
```

Comment:
What about renaming this method to `castCollection` to differentiate between "a collection of DTOs" and a simple array?

Reply:
No problem, I think that increases the consistency of the naming convention in the package.